### DISCUSSION TITLE
Unique Rowy URLs per filter combination

### DISCUSSION THREAD

**Discussion by gregdingle** [AntlerVC/firetable#1065] [Idea / Feature Request]
It would be very useful to get a URL that represents the current filtered view of a table.
1) share with colleagues
2) bookmark for self
3) link to from other tables as a way of show foreign key data (id in one table cell links to filtered view in another table)

For example:

https://rowy.app/p/test/table/documents#filters:documentData.copy_count>0

<img width="890" alt="CleanShot 2023-01-07 at 08 15 00@2x" src="https://user-images.githubusercontent.com/28797/211152637-69fe14ea-c95e-4dd7-af61-c112bbd6890d.png">

**Comment by harinij** [comment]
@gregdingle Good idea. We currently have the ability for a filter to be applied by default to a table for all users, using the section highlighted below. It would be good to have URL based filter as well as you shared.
<img width="669" alt="Screen Shot 2023-01-09 at 6 03 50 pm" src="https://user-images.githubusercontent.com/307298/211255741-040c2894-b0cc-4346-8828-05b2d7c32dbb.png">
